app/controllers/messages_controller.rb

```diff
@@ -0,0 +1,27 @@
+class MessagesController < ApplicationController
+
+  def create
+    message = Message.new(message_params)
+    if message.save
+      if request.xhr?
+        render plain: "<li>#{message.content}</li>"
```

Comment:
I think I'd probably just send the message content down and construct the `<li>` in the JS.